artifactory_test.go

```diff
@@ -4015,3 +4017,37 @@ func TestVcsProps(t *testing.T) {
 	tests.RenamePath(newinnerDotGitPath, filepath.FromSlash((oldinnerDotGitPath + "/gitdata")), t)
 	cleanArtifactoryTest()
 }
+
+// Rename the ".git" dir to it origin name "gitdata" in vcs testdata dir.
+func cleanVcsTestDir(t *testing.T) {
```

Comment:
This function seems to be changing the files which are part of *testdata*, which is part of the project sources.
The tests are not supposed to change the sources.

Reply:
Removed.